molo/profiles/templates/registration/login.html

```diff
@@ -2,27 +2,33 @@
 {% load i18n %}
```

Comment:
Please apply the things that I've been asking you to do in `change_password` in this file as well.